src/core/probe/kernel/bpf/include/common.h

```diff
@@ -5,15 +5,11 @@
 #include <bpf/bpf_core_read.h>
 #include <bpf/bpf_helpers.h>
 
+#include <retis_context.h>
 #include <events.h>
 #include <helpers.h>
 #include <packet-filter.h>
-
-enum kernel_probe_type {
-	KERNEL_PROBE_KPROBE = 0,
-	KERNEL_PROBE_KRETPROBE = 1,
-	KERNEL_PROBE_TRACEPOINT = 2,
-};
+#include <skb-tracking.h>
```

Comment:
do we have / need a guideline regarding casing of headers? I see both kebab and snake casing